source/_cookbook/simulation-mode.py

```diff
@@ -38,21 +38,22 @@
 
 print_summary(count([det]))
 print_summary(scan([det], motor, 1, 3, 3))
-print_summary(outer_product_scan([det], motor1, 1, 3, 3, motor2, 1, 2, 2, False))
+print_summary(outer_product_scan([det], motor1, 1, 3, 3, motor2, 1, 2, 2,
```

Comment:
This should be `grid` now I think.

Reply:
Yeah, and now `print_summary` warns: it should be `summarize_plan`. (Sorry for the annoying name changes! Now that we have 1.0 I promise they will diminish.)

Reply:
@tacaswell yes you're right thanks for the information!
(https://github.com/NSLS-II/bluesky/blob/master/bluesky/plans.py#L1420)
@danielballan thanks for this information as well, tested on my machine and updated.